roles/ceph-rgw/tasks/check_mandatory_vars.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Having a check in `roles/ceph-common/tasks/checks/check_mandatory_vars.yml` is sufficient, so you can remove this one.